package.json

```diff
@@ -103,6 +103,7 @@
     "audit": "yarn npm audit --recursive --environment production --severity moderate"
   },
   "resolutions": {
+    "@metamask/assets-controllers@^9.2.0": "npm:@metamask-previews/assets-controllers@12.0.0-preview.56ca173",
```

Comment:
TODO: Rename this PR to `bump @metamask/assets-controllers` after this is merged and released